base/services/nfsd/nfs41_daemon.c

```diff
@@ -163,10 +165,23 @@ typedef struct _nfsd_args {
 static bool_t check_for_files()
 {
     FILE *fd;
+#ifdef __REACTOS__
+    char config_path[MAX_PATH];
+
+    GetSystemDirectoryA(config_path, ARRAYSIZE(config_path));
+    strcat(config_path, "\\drivers\\etc\\netconfig");
```

Comment:
Check GetSystemDirectory return and use string safe functions please

Reply:
OK.